cmd/client-s3.go

```diff
@@ -1318,6 +1350,11 @@ func (c *S3Client) Stat(isIncomplete, isFetchMeta, isPreserve bool, sse encrypt.
 		return content, nil
 	}
 
+	// If the request is for incomplete upload stat, handle it here.
+	if isIncomplete {
+		return c.statIsComplete(bucket, object)
+	}
+
 	// The following code tries to calculate if a given prefix/object does really exist
```

Comment:
Can we modify this comment as per the current change

Reply:
> Can we modify this comment as per the current change

Nothing is supposed to change in the comment, are you referring to something specific ?